proto/slinky/mm2/v1/params.proto

```diff
@@ -8,4 +8,8 @@ message Params {
   // MarketAuthorities is the list of authority accounts that are able to
   // control updating the marketmap.
   repeated string market_authorities = 1;
+
+  // Admin is an address that can remove addresses from the MarketAuthorities
+  // list. Only governance can add to the MarketAuthorities or change the Admin.
+  string admin = 2;
```

Comment:
wym `Only governance`? the gov module account is a possible value for this parameter, no?

Reply:
No, this parameter could be any address